xdebug_stack.c

```diff
@@ -1383,6 +1402,19 @@ function_stack_entry *xdebug_add_stack_frame(zend_execute_data *zdata, zend_op_a
 	return tmp;
 }
 
+void xdebug_remove_stack_frame(function_stack_entry *fse) {
```

Comment:
New line after the closing `)` and before `{` please. I would probably also rename the function to `xdebug_frame_unlock_objects`, so that you don't need the comment in line 1407/1408.

Reply:
Okay for the code style, but I would keep the method name: we call the `xdebug_add_stack_frame` at 2 places, and the "GC-locking" thing is done in that method. So calling a `xdebug_remove_stack_frame` seems natural IMO (since we added it) while calling a `xdebug_frame_unlock_objects` means we know that the add frame method does the locking (plus, if the add frame does some extra stuff in the future, then its reverse can be added to the remove stack, .while it wouldn't fit in the renamed "unlock" method)
If you agree on that, I'll leave the current method name `xdebug_remove_stack_frame`